monitor-stddev.py

```
@@ -0,0 +1,724 @@
+    with open(filename_output_png, 'wb') as png_file:
+        png_file.write(png_image_data.getvalue())
+
+def build_request_headers(user_agent: Optional[str], authorization: Optional[str]) -> Dict[str, str]:
+    headers: Dict[str, str] = {}
+    if user_agent:
+        headers['User-Agent'] = user_agent
+    if authorization:
+        headers['Authorization'] = authorization
+    return headers
+
+def monitor_website(
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)